### ISSUE TITLE
BoundsError on simple SDP

### ISSUE THREAD

**Issue by mpf** [JuliaOpt/JuMP.jl#1041]
Here is a simple SDP:
```julia
    using SCS, JuMP
    m = Model(solver=SCSSolver())
    @variable(m, X[1:3,1:3], SDP)
    @objective(m, Min, trace(X))
    solve(m)
```
Here's the error that's generated:
```julia
ERROR: BoundsError: attempt to access 0-element Array{Float64,1} at index [1]
 in fillConicDuals(::JuMP.Model) at /Users/mpf/.julia/v0.5/JuMP/src/solvers.jl:131
 in #solve#114(::Bool, ::Bool, ::Bool, ::Array{Any,1}, ::Function, ::JuMP.Model) at /Users/mpf/.julia/v0.5/JuMP/src/solvers.jl:204
 in solve(::JuMP.Model) at /Users/mpf/.julia/v0.5/JuMP/src/solvers.jl:148
```

Looking at the output from SCS, it appears that the actual solve was correct. This was run using Julia 0.5 and JuMP v0.17.0.

**Comment by mlubin** [comment]
Seems like a bug. @blegat

**Comment by blegat** [comment]
Fixed by #1042 which is part of JuMP v0.17.1. You will be able to just do `Pkg.update()` once [this PR is merged](https://github.com/JuliaLang/METADATA.jl/pull/9744).
